examples/AzureSDKSmokeTestSwiftPM/AzureSDKSmokeTestSwiftPM.xcodeproj/project.pbxproj

```diff
@@ -0,0 +1,360 @@
+// !$*UTF8*$!
```

Comment:
Where does this get build and ran? Also how does it actually pull in the swift package references? Perhaps I'm missing something but I would expect to pull them in somewhere.

Reply:
I think I had that in there for testing. For now I will remove it and re-add it in a later PR, as it should be part of CI the way the CocoaPods smoke test is.